cli/root.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/odpf/compass/internal/client"
+	"github.com/odpf/salt/cmdx"
+	"github.com/spf13/cobra"
+)
+
+func New(cfg *Config) *cobra.Command {
+	cliConfig = cfg
```

Comment:
I see we are using a global variable for config but also in the `server start` command loading configs again?